moveit_ros/visualization/planning_scene_rviz_plugin/src/planning_scene_display.cpp

```diff
@@ -583,14 +583,17 @@ void PlanningSceneDisplay::sceneMonitorReceivedUpdate(
 void PlanningSceneDisplay::onSceneMonitorReceivedUpdate(
     planning_scene_monitor::PlanningSceneMonitor::SceneUpdateType /*update_type*/)
 {
-  bool old_state = scene_name_property_->blockSignals(true);
   getPlanningSceneRW()->getCurrentStateNonConst().update();
-  scene_name_property_->setStdString(getPlanningSceneRO()->getName());
-  scene_name_property_->blockSignals(old_state);
-
+  QMetaObject::invokeMethod(this, "setSceneName", Qt::QueuedConnection,
```

Comment:
```suggestion
  QMetaObject::invokeMethod(this, &PlanningSceneDisplay::setSceneName, Qt::QueuedConnection,
```
probably doesn't work due to the argument right?

Reply:
Yeah, tried this first. But invokeMethod doesn't support methods, only lambdas.